staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -36,30 +45,29 @@ type Config struct {
 	// Something that can list and watch your objects.
 	ListerWatcher
 
-	// Something that can process your objects.
+	// Something that can process a popped Deltas.
 	Process ProcessFunc
 
-	// The type of your objects.
+	// ObjectType is an example object of the type this controller is
+	// expected to handle.  Only the type needs to be right, except
+	// that when that is `unstructured.Unstructured` the object's
+	// `"apiVersion"` must also be right.
```

Comment:
Is it only apiVersion or also kind?
[I don't know - I'm asking]

Reply:
Oh, yes, "kind" has to match too.